cmd/cli/serve/ucan.go

```diff
@@ -257,6 +265,7 @@ func startServer(cmd *cobra.Command, _ []string) error {
 		storage.WithIdentity(id),
 		storage.WithBlobstore(blobStore),
 		storage.WithAllocationDatastore(allocDs),
+		storage.WithAcceptanceDatastore(acceptDs),
```

Comment:
This server can never be used with PDP, does this change make sense to include here?